### ISSUE TITLE
FloatNSplit component

### ISSUE THREAD

**Issue by BaxterOttoman** [Yellow-Dog-Man/Resonite-Issues#2065]
### Is your feature request related to a problem? Please describe.

There is a component that composes FloatN values from two to four inputs. There is no component option that goes the other way around and lets you extract float values from a FloatN and use them to drive float fields.

### Describe the solution you'd like

The desired behavior would be have a target FloatN value and N number of float targets, labeled Float X Target, Float Y Target, etc, as needed. The FloatN field would take it's referenced value and drive the targeted float fields wherever filled in.

### Describe alternatives you've considered

Flux provides alternatives, but they take several nodes. Swizzle drivers are applicable because while they can freely interchange Floats of 2, 3, 4, Q, they can't bring down single values.

### Requesters

Baxter, ShadowX

**Comment by lxw404** [comment]
It might be that the single value case of the swizzle drivers is just missing which should solve this.

**Comment by 5H4D0W-X** [comment]
> It might be that the single value case of the swizzle drivers is just missing which should solve this.

The bit with swizzle drivers is that inputs and outputs can be mixed and matched, but that doesn't really apply with single values. It would also take 1 to 3 extra components to decompose the entire value when using a FloatNToFloatSwizzleDriver since each instance of the component could only output one value (assuming it follows the same structure as the other swizzles).
I think the components outlined in the issue would solve the problem more efficiently.

**Comment by lxw404** [comment]
Oh so you want a way to individually drive N outputs from an N-dimensional input?

**Comment by BaxterOttoman** [comment]
Yeah, just like how the FloatN Drivers reference N number of float fields and drive one target, these take one FloatN field and drive N numers of floats.

**Comment by epicEaston197** [comment]
Not too familiar but can't you just use the unpack and pack nodes?
Resonite devs really don't want to have duplicated functionality and would likely won't add this because of that reason

**Comment by JackTheFoxOtter** [comment]
> Not too familiar but can't you just use the unpack and pack nodes?
Resonite devs really don't want to have duplicated functionality and would likely won't add this because of that reason

There's already a component to do it the other way around, I don't see why this one would be different.

I know @mpmxyz actually had a use-case for this not long ago and was looking for a component to do this. I'd like to see this added as well.

**Comment by BaxterOttoman** [comment]
Yeah, looking at components that already exist, I think this functionality lies in line with the philosophy of those components. They do one thing directly based directly on another value with the intention of that being the whole thing.

**Comment by yoshiyoshyosh** [comment]
Is this the same request as #495?
